app/queries/search_results_query.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+CONDITION_URL_EQUALS = ':url = ANY (adwords_top_urls) OR :url = ANY (non_adwords_urls)'
```

Comment:
This condition should be inside the class? 🤔 Reference on [Compass](https://nimblehq.co/compass/development/code-conventions/ruby/ruby-on-rails/#convention-31)